content/dstu2/observation.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Observation | FHIR DSTU 2 BALLOT API
+---
+
+# Observation
+
+* TOC
+{:toc}
+
+## Search
+
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Parameters
+
+ Name            | Required?| Type                                                           | Description
+-----------------|----------|----------------------------------------------------------------|-------------------------------------------------------
+`patient`        | Either this or `subject` |[`reference`](http://hl7.org/fhir/dstu2/search.html#reference)| The subject that the observation is about (if patient). EG: `123456`
```

Comment:
In @davischristi 's DiagnosticReport PR he wrote these as "This or subject".  We should use a consistent pattern, I have no real preference on which one

https://github.com/cerner/fhir.cerner.com/pull/42/files#diff-a1330e0905fd78f90fc132493e2caf4eR20